mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -185,16 +199,21 @@ def _SendConfiguration(self, configuration, fireandforget=False, timeout=2.0, ch
     #
 
     def StartObjectDetectionTask(self, taskId=None, systemState=None, visionTaskParameters=None, timeout=2.0, **ignoredArgs):
-        # type: (Optional[types.SystemState], Optional[types.visionTaskObjectDetectionParametersSchema], float, Dict) -> Optional[Dict]
+        # type: (Optional[str], Optional[types.SystemState], Optional[types.visionTaskObjectDetectionParametersSchema], float, Dict) -> Optional[Dict[str, str]]
```

Comment:
**ignoredArgs -> is that a dict or Optional[Dict] ?

Reply:
Thank you for pointing it out! I was just following the existing code, but it wasnt a good approach.

The type here should actually be Any. See https://stackoverflow.com/questions/37031928/type-annotations-for-args-and-kwargs .

I've also checked it with Pylance. It discovered multiple other type issues. Fixed them now. PTAL.